scripts/plotting/polar_map.py

```diff
@@ -366,5 +411,22 @@ def polar_map(adfobj):
 ##############
 #END OF `polar_map` function
 
+#########
+# Helpers
+#########
+
+def _get_climo_yrs(cam_ts_loc):
```

Comment:
If we save the start and end years as part of the ADF object in `adf_info.py` then this function and related code isn't needed.